src/Cache/LoggingCachePool.php

```diff
@@ -155,4 +157,27 @@ public function getCalls()
     {
         return $this->calls;
     }
+
+    /**
+     * Get a string to represent the value.
+     *
+     * @param mixed $value
+     *
+     * @return string
+     */
+    private function getValueRepresentation($value)
+    {
+        $type = gettype($value);
+        if (in_array($type, ['boolean', 'integer', 'double', 'string', 'NULL'])) {
+            $rep = $value;
+        } elseif ($type === 'array') {
+            $rep = json_encode($value);
+        } elseif ($type === 'object') {
+            $rep = get_class($value);
+        } else {
+            $rep = sprintf('<DATA:%s>', $type);
```

Comment:
You are talking about if $type=='object'? Yeah, why not.

On line 178 we do only have 'resource' and some other thing we dont want to display.